packages/relay/src/lib/services/ethService/transactionService/TransactionService.ts

```diff
@@ -106,9 +106,7 @@ export class TransactionService implements ITransactionService {
     transactionIndex: string,
     requestDetails: RequestDetails,
   ): Promise<Transaction | null> {
-    if (this.logger.isLevelEnabled('trace')) {
-      this.logger.trace(`getTransactionByBlockHashAndIndex(hash=${blockHash}, index=${transactionIndex})`);
-    }
+    this.logger.trace('getTransactionByBlockHashAndIndex(hash=%s, index=%s)', blockHash, transactionIndex);
```

Comment:
I believe we can remove most of the other guards in this file as well except the one in `validateRawTransaction` and `sendRawTransactionErrorHandler`